src/collections/members/olushesi-toheeb/index.mdx

```diff
@@ -0,0 +1,11 @@
+---
+layout: person
+name: Olushesi Toheeb
```

Comment:
has this been on layer5-ng too?

Reply:
@OlushesiToheeb just now being created in this PR, yes.